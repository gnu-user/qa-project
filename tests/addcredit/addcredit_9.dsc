TITLE
    Privileged Username Input Validation (Valid Input)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the addcredit 
    transaction. When prompted, enter a valid decimal input less than the 
    maximum limit (1000.00) for the amount of credit to add. When prompted, 
    enter a valid username. Verify that it is accepted without error.
DEPENDENCIES
    addcredit_7
