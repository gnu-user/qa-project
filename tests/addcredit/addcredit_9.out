Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the amount of credit to add: 
Enter the username to which credit is being added: 
Credit added successfully.
> 